src/styles/mixins/_list-table.scss

```
@@ -0,0 +1,37 @@
+@use "variables/colors";
+@use "@angular/material/core/style/elevation" as mat-elevation;
+
+// Shared look of the admin list tables (user list, roles list, permission
+// matrix) so the three cannot drift apart. Each component @includes only the
+// parts it needs.
+
+@mixin container-and-table {
+  .table-container {
+    position: relative;
+    @include mat-elevation.elevation(2);
+    background-color: white;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)